app/views/layouts/application.html.erb

```diff
@@ -7,34 +7,45 @@
   
 </head>
 <body>
-    <div class="container">
-      <% flash.each do |message_type, message| %>
-        <div class="alert alert-<%= message_type %>"><%= message %></div>
-      <% end %>
+  <div class="container">
+    <% flash.each do |message_type, message| %>
```

Comment:
Be careful doing something like this. While the idea is great, what if someone in the future does something like flash[:foo]="bar"?

Instead, why not only look for a select set of keys from the flash?